adminutils/adminutils.py

```diff
@@ -223,6 +240,7 @@ async def emote_steal(self, ctx, name: str, message_id: discord.Message, *roles:
                     else None,
                 ),
             )
+            await ctx.send(_(f"{e} created with the name `{e.name}`."))
```

Comment:
Not sure about this part. Others commands doesn't have any sort of message on success. Only tick reaction.

Reply:
I think sending this allows for the author to see the immediate effect of the command and is better than the amiguous tick. I'll update all the commands to that.